tools-benchmarks/src/main/scala/scala/scalanative/linker/benchmarks/TestSuiteLinkerBench.scala

```diff
@@ -0,0 +1,43 @@
+package scala.scalanative.linker.benchmarks
+
+import java.nio.file.{Path, Paths, Files}
+import java.nio.charset.StandardCharsets
+import java.util.concurrent.TimeUnit
+
+import scala.scalanative.build
+import scala.scalanative.internal.build.TestSuiteBuildInfo
+
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.annotations.Mode.SampleTime
+
+@State(Scope.Benchmark)
+abstract class TestSuiteLinkerBench
```

Comment:
While you are at it can you also please add:
1. TestSuiteInterflowBench. Benchmark just interflow optimization + in-memory reachability that follows, i.e., `build.ScalaNative.optimize` call. 
2. TestSuiteCodeGenBench. Benchmark just the `build.ScalaNative.codegen` call.

Reply:
Actually, it would also be good to add total SN pipeline time as `TestSuiteBuildBench` which calls `build.Build.build`.